src/adv.py

```diff
@@ -54,7 +100,47 @@
     },
 """
 
+
 # Write a class to hold player information, e.g. what room they are in currently
+class Player:
+    def __init__ (self, startRoom, playerHolding, playerName, playerSkills)
+    self.currentRoom = startRoom
+    self.inventory = playerHolding
```

Comment:
Also this portion can be a list "array" since the player may have multiple items in the inventory. However the player probably is only capable of equipping 1 or 2 of those items on each hand...